docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:10.8.0
```

Comment:
@Djuki First thank you for your contribution!!! We are very happy =)
What do you think of using some server image instead?